src/schemas/21P-530/schema.js

```diff
@@ -124,7 +119,16 @@ let schema = {
     },
     amountGovtContribution: {
       type: 'number'
-    }
+    },
+    claimantEmail: {
+      type: 'string',
+      format: 'email'
+    },
+    'view:claimantEmailConfirmation': {
```

Comment:
Thanks @lihanli, @J-Quag and @stacyhlinn provided feedback on [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/5752#issuecomment-308832233), indicating that the email should be checked as in EDU. Please let me know if I should approach this differently.